src/test/java/seedu/address/testutil/CustomerBuilder.java

```diff
@@ -109,16 +109,12 @@ public CustomerBuilder withDateOfBirth(String dateOfBirth) {
         this.dateOfBirth = new DateOfBirth(dateOfBirth);
         return this;
     }
-
     /**
-     * Parses the {@code carsOwned} into a {@code Map<Car, CoeExpiry>} and set it to the {@code Customer} that we are
-     * building.
-     *
-     * @param carsOwned
-     * @return
+     * Adds a new pairing of {@code Car} and {@code CoeExpiry} to the
+     * {@code Customer} that we are building.
      */
-    public CustomerBuilder withCarsOwned(Map<Car, CoeExpiry> carsOwned) {
```

Comment:
why did you delete this? we want to support multiple cars owned by the customer, all the code is written like that.